src/LuaTransformer.ts

```diff
@@ -3362,6 +3388,18 @@ export class LuaTransformer {
 
                     result.push(assignmentStatement);
                     break;
+                case ts.SyntaxKind.SpreadElement:
+                    if (index !== node.elements.length - 1) break;
```

Comment:
Should we just ignore this or should this throw an error? Is this valid TS?

Reply:
It's invalid TS and that's exactly why we shouldn't throw there. TypeScript already provides a diagnostic, so there's no point to break compilation for user.

Reply:
I agree.